unum/Makefile

```diff
@@ -19,7 +19,7 @@ PKG_VERSION:=v2
 PKG_LICENSE:=Apache-2.0
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=c3b619c0e33723c2796bba542edd73b762792d2d
+PKG_SOURCE_VERSION:=f2ec84d55bf9c61f1d2c1d2fda1acc28628fdd13
 PKG_MIRROR_HASH:=d4f03b3b39314b0018248784d76b5754ee4659764c3c92b31505930ca0c495f5
```

Comment:
I think this is going to fail the download check if you don't also update the mirror hash.  I usually do this:
```
# edit feeds/minim/unum/Makefile with the new PKG_SOURCE_VERSION
rm dl/unum-v2.tar.xz
make download -jX
make package/unum/check FIXUP=1
# feeds/minim/unum/Makefile should have a fixed up PKG_MIRROR_HASH now
```